docs/diagnostics/PX1032.md

```diff
@@ -0,0 +1,50 @@
+# PX1032
+This document describes the PX1032 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                 | Type  | Code Fix    | 
+| ------ | ------------------------------------------------- | ----- | ----------- | 
+| PX1032 | DAC properties cannot contain method invocations. | Error | Unavailable | 
+
+## Diagnostic Description
+DACs cannot contain method invocations. DACs are the classes that are used to read and write data. Therefore, for appropriate program architecture and design, DACs cannot contain any application logic. 
+
+To fix the issue, you should remove method invocations from the DAC.
+
+The diagnostic ignores invocations of the following methods:
+
+ - Static methods inside DAC fields (that is, property getters and setters)
+ - Methods declared on the system types, such as `string`, `int`, `DateTime`, `Guid`, and `TimeSpan`
```

Comment:
the list of the system types:
bool
byte
short
int
long
decimal
float
double
string
Guid
DateTime
TimeSpan
Enum
Nullable<T>

Reply:
fixed